libs/entitlement/qa/entitlement-test-plugin/src/main/java/org/elasticsearch/entitlement/qa/test/VersionSpecificNetworkChecks.java

```diff
@@ -19,7 +19,7 @@
 
 class VersionSpecificNetworkChecks {
     static void createInetAddressResolverProvider() {
-        var x = new InetAddressResolverProvider() {
+        new InetAddressResolverProvider() {
```

Comment:
In my IDE I got an "unused variable" warning but not an "ignored return value" warning. 🤷